go.mod

```diff
@@ -58,7 +58,9 @@ require (
 	k8s.io/apimachinery v0.21.0
 	k8s.io/apiserver v0.21.0
 	k8s.io/client-go v0.21.0
+	k8s.io/cloud-provider v0.21.0
 	k8s.io/component-base v0.21.0
+	k8s.io/klog v0.3.0
```

Comment:
We have switched to klog/v2, is this imported on purpose?

Reply:
No, by mistake